src/projections_site/util.py

```diff
@@ -0,0 +1,40 @@
+"""Utilities for parsing CRS"""
+
+from typing import Any
+
+import pyproj
+
+
+def get_datum(crs: pyproj.CRS) -> Any:
+    """Returns datum name"""
+    try:
+        datum_name = crs.datum
```

Comment:
```suggestion
def get_datum(crs: pyproj.CRS) -> string:
    """Returns datum name"""
    try:
        datum_name = crs.datum
```

I think we should avoid `Any` wherever we can, because we know we want these functions to always return strings. Where `pyproj` lacks type information, we can always coerce or cast, e.g.: `datum_name = string(crs.datum)` or `datum_name = crs.datum as string`.

Reply:
I had the return type as str and mypy won't pass it so I had to change to Any.  Any suggestion to address it? thanks!

Reply:
Yeah! `Any` should always be seen as a last resort; it effectively turns off typechecking for that thing. Instead, try using `reveal_type(something)` to see what type Mypy is inferring for an object, e.g. `reveal_type(crs.datum)`. If that returns `Any`, then Mypy needs more information to understand what that thing is. We can use casting (`as str`) to tell Mypy "believe me, that's a string", or coercion (`str(something)`) to convert the value into a string.

It looks like mypy only complained about one line, but we have `Any` three times here, so I think we only need to do coercion/casting in one place.

Reply:
thanks! i will try that.